app/components/primer/close_button.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module Primer
+  # Use close button to render a `×` without default button styles.
+  class CloseButton < Primer::Component
+    DEFAULT_TYPE = :button
+    TYPE_OPTIONS = [DEFAULT_TYPE, :submit].freeze
+
+    # @example Default
+    #   <%= render(Primer::CloseButton.new) %>
+    #
+    # @param type [Symbol] <%= one_of(Primer::CloseButton::TYPE_OPTIONS) %>
+    # @param system_arguments [Hash] <%= link_to_system_arguments_docs %>
+    def initialize(type: DEFAULT_TYPE, **system_arguments)
+      @system_arguments = system_arguments
+      @system_arguments[:tag] = :button
+      @system_arguments[:type] = fetch_or_fallback(TYPE_OPTIONS, type, DEFAULT_TYPE)
+      @system_arguments[:classes] = class_names(
+        "close-button",
+        system_arguments[:classes]
+      )
+    end
+
+    def call
+      render(Primer::BaseComponent.new(**@system_arguments)) do
```

Comment:
This would be a nice place to use `with_content` 😄

Reply:
soon!